java/sqlcommenter-java/src/main/java/com/google/cloud/sqlcommenter/schibernate/SCHibernate.java

```diff
@@ -14,16 +14,11 @@
 
 package com.google.cloud.sqlcommenter.schibernate;
 
+import com.google.cloud.sqlcommenter.threadlocalstorage.SpanContextMetadata;
 import com.google.cloud.sqlcommenter.threadlocalstorage.State;
-import io.opencensus.trace.SpanContext;
-import io.opencensus.trace.Tracer;
-import io.opencensus.trace.Tracing;
 import org.hibernate.resource.jdbc.spi.StatementInspector;
 
 public class SCHibernate implements StatementInspector {
-
-  private static final Tracer tracer = Tracing.getTracer();
```

Comment:
I believe removing this from this call site makes it expensive. Why did we remove this code?

Reply:
Yes, you're right. I had moved it into the method to separate OC and OT implementation. Moved back to declare it in the class to reduce computation now